### PR TITLE
[GITHUB-2] Inital code for Elastic Filebeat

### PR DESCRIPTION
**Pull request by wojtek-oledzki**

### CODE REVIEW

File: tasks/build.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  become: yes
+  apt_repository:
+    repo: "deb https://artifacts.elastic.co/packages/6.x/apt stable main"
+    state: present
+    update_cache: yes
+
+- name: Install Filebeat
+  become: yes
+  apt:
+    name: filebeat
```
Comment (nathanwebsterdotme) [inline tasks/build.yml]: We should be able to specify a specific version here, not just default to latest

File: tasks/build.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+
+# Elastic Filebeat is might be extracted to it's own role later on.
```
Comment (nathanwebsterdotme) [inline tasks/build.yml]: Remove this

File: tasks/configure.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+
+# Elastic Filebeat is might be extracted to it's own role later on.
```
Comment (nathanwebsterdotme) [inline tasks/configure.yml]: Remove this

**Review by nathanwebsterdotme — APPROVED** [review]

**Comment by lobsterdore** [comment]
Shouldn't this be going into develop instead of master?

File: defaults/main.yml [hunk 4]
```diff
@@ -1 +1,6 @@
 ---
+
+sansible_filebeat_configuration: {}
+sansible_filebeat_version: 6.6.1
+sansible_filebeat_repo_key_id: D88E42B4
+sansible_filebeat_repo_keyserver: keyserver.ubuntu.com
```
Comment (lobsterdore) [inline defaults/main.yml:5]: Needs to be alpha ordered.

Comment (wojtek-oledzki) [reply defaults/main.yml:5]: 👍

**Review by lobsterdore — APPROVED** [review]
LGTM 👍

**Pull request merged**